src/os/feature/feature.js

```diff
@@ -29,6 +29,7 @@ const DynamicFeature = goog.require('os.feature.DynamicFeature');
 const fields = goog.require('os.fields');
 const {reduceExtentFromGeometries} = goog.require('os.fn');
 const geo = goog.require('os.geo');
+const {validate: jstsValidate} = goog.require('os.geo.jsts');
```

Comment:
To encourage consistent naming, named exports should not be renamed on import. Either use the original name, or import the full module exports using a name based on the import path (ie, `osGeoJsts`).

https://google.github.io/styleguide/jsguide.html#es-module-imports